src/core/crypto.c

```diff
@@ -1720,6 +1720,11 @@ QuicCryptoCustomCertValidationComplete(
             QuicCryptoGetConnection(Crypto),
             "Custom cert validation succeeded");
         QuicCryptoProcessDataComplete(Crypto, Crypto->PendingValidationBufferLength);
+
+        if (QuicRecvBufferHasUnreadData(&Crypto->RecvBuffer)) {
+            // More data was received while waiting for user to perform the validation.
```

Comment:
Please update to the style we use for comments.
```suggestion
            //
            // More data was received while waiting for user to perform the validation.
            //
```
Same for the function below.